openapi/cosmos_v2.yaml

```diff
@@ -1314,26 +1349,63 @@ components:
 
 # SDTM Dataset Specialization Domains
     sdtmDatasetSpecializationDomains:
+      type: object
```

Comment:
@nhaydel For the domains we could have title: SDTM Dataset Specialization Domain (VS), and for the categories: title: Biomedical Concepts Category (Vital Sign). Is that what you mean?
![image](https://github.com/cdisc-org/COSMoS/assets/11705643/86190157-ae9e-4c8c-8219-33ddd5b4c665)

Reply:
Not quite, I mean having the name like "Vital Signs" and "DM" in the link. Or having each item of the categories array and domains array be its own object with a name and a self link. That way any potential clients don't have to parse the link title to get the name of the object

Reply:
Makes sense. I will create an update for review. Thanks!

Reply:
I update so that each item of the categories array and domains array is its own object with a name and a self link.
![image](https://github.com/cdisc-org/COSMoS/assets/11705643/47b0b89d-efc4-4f8d-b343-e970ae605d47)
![image](https://github.com/cdisc-org/COSMoS/assets/11705643/e4fe1413-3576-4018-8f40-cede215a4a2e)

Reply:
Fixed it:
![image](https://github.com/cdisc-org/COSMoS/assets/11705643/6d4fcfb1-29fc-4ca3-9c0c-37945a5501b4)
![image](https://github.com/cdisc-org/COSMoS/assets/11705643/dbb5d370-0751-41de-af65-0463d0a38ca4)